slack-plugin/src/main/kotlin/slack/gradle/StandardProjectConfigurations.kt

```diff
@@ -815,43 +837,14 @@ internal class StandardProjectConfigurations {
       }
     }
 
-    pluginManager.withPlugin("io.gitlab.arturbosch.detekt") {
-      // Configuration examples https://arturbosch.github.io/detekt/kotlindsl.html
-      configure<DetektExtension> {
-        toolVersion =
-          slackProperties.versions.detekt ?: error("missing 'detekt' version in version catalog")
-        rootProject.file("config/detekt/detekt.yml")
-        slackProperties.detektConfigs?.let { configs ->
-          for (configFile in configs) {
-            config.from(rootProject.file(configFile))
-          }
-        }
-
-        slackProperties.detektBaseline?.let { baselineFile ->
-          baseline =
-            if (globalConfig.mergeDetektBaselinesTask != null) {
-              tasks.withType<DetektCreateBaselineTask>().configureEach {
-                globalConfig.mergeDetektBaselinesTask.configure { baselineFiles.from(baseline) }
-              }
-              file("$buildDir/intermediates/detekt/baseline.xml")
-            } else {
-              file(rootProject.file(baselineFile))
-            }
-        }
-      }
-    }
-
     pluginManager.withPlugin("org.jetbrains.kotlin.jvm") {
-      onKotlinPluginApplied()
```

Comment:
> Use the new kotlin base plugin as a hook

Question for my understanding. Why did we need to invoke this lambda multiple times before, and we don't need it anymore?

Reply:
before we had to hook in both the jvm and android plugins, but now we get the "base" plugin as a common hook for these common configs. The android plugin has a similar tool